extensions/positron-r/language-configuration/r-language-configuration.json

```diff
@@ -55,6 +55,13 @@
 			"action": { "indent": "none", "appendText": "#* " }
 		},
 
+		// Regular comments.
```

Comment:
Should we add the info re: "order matters" in this comment? So no well-meaning future us moves this without good reason.